scripts/upgrade_instance_class.rb

```diff
@@ -0,0 +1,202 @@
+#!/usr/bin/env ruby
+
+require "aws-sdk-rds"
+require "versionomy"
```

Comment:
Where does versionomy come from? Is it already included in the repo's Gemfile?

Reply:
In `manifests/cf-manifest/spec/manifest/release_versions_spec.rb` we appear to use `Gem::Version`

Reply:
Code now uses Gem::Version